src/main/java/com/skhu/luxuryshop/user/controller/UserController.java

```diff
@@ -0,0 +1,32 @@
+package com.skhu.luxuryshop.user.controller;
+
+import com.skhu.luxuryshop.user.dto.UserSignupDto;
+import com.skhu.luxuryshop.user.service.UserSignupService;
+import lombok.NoArgsConstructor;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/users")
+public class UserController {
+    private final UserSignupService userSignupService;
+
+    //이메일 중복검사
+    @GetMapping("/email")
+    public ResponseEntity<Boolean> isDuplicatedEmail(@RequestBody String Email) {
+        boolean checkResult = userSignupService.isDuplicatedEmail(Email);
+        return new ResponseEntity(checkResult, HttpStatus.OK);
+    }
+
+    //회원가입
+    @PostMapping("/signup")
+    public ResponseEntity<String> signup(@RequestBody @Valid UserSignupDto userSignupDto) {
+        userSignupService.save(userSignupDto);
+        return new ResponseEntity("회원가입 성공", HttpStatus.OK);
```

Comment:
POST 매핑으로 새로운 데이터를 저장했을 경우에는 200 OK도 가능하지만 
보통 201 created 상태 코드를 씁니다!
그리고 새로 생성 된 리소스를 반환받을 수 있는 uri를 header에 넣어줍니다!

응답한 데이터를 명확하게 표현하기 위한 ststus code 중에하나인 201 code도 공부해보길 추천해요!

userSignupService.save를 조금 수정하고 이런식으로 수정하면 될 거예요.
```suggestion
        return ResponseEntity.created(URI.create("/" + savedUser.getId())).build();
```